tests-ng/plugins/kernel_configs.py

```diff
@@ -0,0 +1,54 @@
+import glob
+import platform
+
+import pytest
+
+
+@pytest.fixture(scope="session")
+def kernel_config_paths():
+    """Get all available /boot/config-* files."""
+    configs = sorted(glob.glob("/boot/config-*"))
+    assert configs, "No kernel config files found under /boot/config-*"
```

Comment:
we should not have asserts in plugins

returning an empty list and asserting in the test would be fine imo

btw we have this test in the old framework to assert that there is only one kernel https://github.com/gardenlinux/gardenlinux/blob/194499e69d622c252bed70c3b00fb7759bce4338/features/cloud/test/test_only_one_kernel.py#L5